guides/source/active_record_migrations.md

```diff
@@ -588,15 +731,23 @@ While it's not required, you might want to add foreign key constraints to
 add_foreign_key :articles, :authors
 ```
 
-This [`add_foreign_key`][] call adds a new constraint to the `articles` table.
-The constraint guarantees that a row in the `authors` table exists where
-the `id` column matches the `articles.author_id`.
+The [`add_foreign_key`][] call adds a new constraint to the `articles` table.
+The constraint guarantees that a row in the `authors` table exists where the
+`id` column matches the `articles.author_id` to ensure all reviewers listed in
```

Comment:
From "to ensure" to the period, I feel it's redundant and the sentence is more effective without that portion.